fp3/fp3decimal_test.go

```diff
@@ -437,3 +437,22 @@ func TestDecimalMemoryLayout(t *testing.T) {
 		t.Error(a, v)
 	}
 }
+
+func TestDecimal_Compare(t *testing.T) {
+	a := fp3.FromFloat(1.123)
+
+	// a > b
+	if a.Compare(fp3.FromFloat(1.122)) != 1 {
+		t.Error(a, fp3.FromFloat(1.122), a.Compare(fp3.FromFloat(1.122)))
+	}
+
+	// a < b
+	if a.Compare(fp3.FromFloat(1.124)) != -1 {
+		t.Error(a, fp3.FromFloat(1.124), a.Compare(fp3.FromFloat(1.124)))
+	}
+
+	// a == b
+	if a.Compare(fp3.FromFloat(1.123)) != 0 {
+		t.Error(a, fp3.FromFloat(1.123), a.Compare(fp3.FromFloat(1.123)))
+	}
```

Comment:
could you use FromString? this supposed to be primary way of constructing such numbers.

* shorter notation
* reusing b
* avoiding repetition of result (should be clear what is going on)

alternatives are welcome, but please keep short + without floats.

```suggestion
	a, _ := fp3.FromString("1.123")
	
	if b, _ := fp3.FromFloat(1.122); a.Compare(b) != 1 {
		t.Error(a, ">", b)
	}

	if b, _ := fp3.FromFloat(1.124); a.Compare(b) != -1 {
		t.Error(a, "<", b)
	}

	if b, _ := fp3.FromFloat(1.123); a.Compare(b) != 0 {
		t.Error(a, "==", b)
	}
```

Reply:
Got it!